make: *** No rule to make target 'gpt-2'.  Stop.

make: *** No rule to make target 'gpt-2'.  Stop.
why

For context : 

When running the [GPT inference example](https://github.com/ggerganov/ggml/blob/master/README.md#gpt-inference-example), from the README, at the step `make -j4 gpt-2 gpt-j`, we get the following error : 

```
make: *** No rule to make target `gpt-2'.  Stop.
```

Any idea how to fix this ?

--- a/README.md
+++ b/README.md
@@ -77,7 +77,7 @@ git clone https://github.com/ggerganov/ggml
 cd ggml
 mkdir build && cd build
 cmake ..
-make -j4 gpt-2 gpt-j
+make -j4 gpt-2-backend gpt-j
 
 # Run the GPT-2 small 117M model
 ../examples/gpt-2/download-ggml-model.sh 117M

